docs/projects/ad5766_sdz/index.rst

```diff
@@ -0,0 +1,271 @@
+.. _ad5766_sdz:
+
+AD5766-SDZ HDL project
+===============================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The :adi:`AD5766`/ :adi:`AD5767` are 16-channel, 16-/12-bit, voltage output
+Digital-to-Analog Converters (DAC). The DAC generates output voltage ranges from
+an external 2.5 V reference. Depending on the span selected, the mid-point of
+the output span can be adjusted allowing for a minimum output voltage as low as
+−20 V or a maximum output voltage of up to +14 V.
+The :adi:`AD5766`/ :adi:`AD5767` have integrated output buffers which can sink
+or source up to 20 mA. This makes the AD5766/AD5767 suitable for Indium Phosphide
+Mach Zehnder Modulator (InP-MZM) biasing applications.
+The part incorporates a power-on reset circuit that ensures that the DAC outputs
+power up to 0V and remain at this level until the output range of the DAC is
+configured. The outputs of all DACs are updated through register configuration,
+with the added functionality of user-selectable DAC channels to be simultaneously
+updated.
+The :adi:`AD5766`/ :adi:`AD5767` require four power supplies. AVCC is the analog
+supply for the low voltage DAC circuitry. AVDD and AVSS are the positive and
+negative high voltage power supplies for the output amplifiers. A VLOGIC supply
+pin is provided to set the logic levels for the digital interface pins.
+The :adi:`AD5766`/ :adi:`AD5767` utilize a versatile 4-wire serial interface
+that operates at clock rates of up to 50 MHz for write mode and up to 10MHz for
+readback and daisy-chain mode, and is compatible with SPIR, QSPI., MICROWIRE. 
+and DSP interface standards.
+The :adi:`AD5766`/ :adi:`AD5767` are available in a 4mm x 4mm WLCSP package
+and operates at the range of -40C to +105C.
+
+Applications:
+  * Mach Zehnder Modulator Bias Control
+  * Analog Output Modules
+  * Process Control
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`EVAL-AD5766 <EVAL-AD5766>`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`AD5766`
+-  :adi:`AD5767`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+-  :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>` on FMC slot
+-  :adi:`SDP-I-FMC <EVAL-SDP-I-FMC>`
+
+Block design
+-------------------------------------------------------------------------------
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Please insert this line here: https://github.com/analogdevicesinc/hdl/blob/main/docs/projects/template/index.rst?plain=1#L90